BigCommerce/Integration/00_Integration.md

```diff
@@ -0,0 +1,36 @@
+# Integration
+
+In this part of the *BigCommerce* documentation, you will find the following chapters:
+
+- [Manage BigCommerce connection](./01_ManageBigCommerceConnection.md)
+   - [Create BigCommerce connection automatically](./01_ManageBigCommerceConnection.md#create-bigcommerce-connection-automatically) 
+   - [Create BigCommerce connection manually](./01_ManageBigCommerceConnection.md#create-bigcommerce-connection-manually)
+   - [Configure BigCommerce connection](./01_ManageBigCommerceConnection.md#configure-bigcommerce-connection)
+   - [Manage ETL mappings](../Integration/01_ManageBigCommerceConnection.md#manage-etl-mappings)
+
+- [Particularities](./02_Particularities.md)  
```

Comment:
BigCommerce particularities?
(wie in Salesforce)

Reply:
Done